AdditionalInlinePostModMenu.user.js

```diff
@@ -1,44 +1,57 @@
 //
-// @include      https://*stackoverflow.com/*
-// @include      https://*serverfault.com/*
-// @include      https://*superuser.com/*
-// @include      https://*askubuntu.com/*
-// @include      https://*mathoverflow.net/*
-// @include      https://*.stackexchange.com/*
-//
-// @exclude      *chat.*
-// @exclude      *blog.*
+// @match        *://*.stackoverflow.com/*
+// @match        *://*.stackexchange.com/*
+// @match        *://*.stackapps.com/*
+// @match        *://*.superuser.com/*
+// @match        *://*.serverfault.com/*
+// @match        *://*.askubuntu.com/*
+// @match        *://*.mathoverflow.net/*
 //
 // @require      https://raw.githubusercontent.com/samliew/SO-mod-userscripts/master/lib/common.js
+// @require      https://unpkg.com/sweetalert/dist/sweetalert.min.js
 // ==/UserScript==
 
-/* globals StackExchange, GM_info */
+/* eslint-disable no-multi-spaces */
+/* global $:readonly             */
+/* global StackExchange:readonly */
+/* global fkey:readonly          */
+/* global swal:readonly          */
 
 'use strict';
 
 // Moderator check
 if (!isModerator()) return;
 
-const superusers = [584192, 366904, 6451573];
-const isSuperuser = superusers.includes(StackExchange.options.user.userId);
+const newlines  = '\n\n';
+const seApiKey  = 'lSrVEbQTXrJ4eb4c3NEMXQ((';
+const fkey      = StackExchange.options.user.fkey;
+const isSO      = location.hostname == 'stackoverflow.com';
+const isSOMeta  = location.hostname == 'meta.stackoverflow.com';
+const isMeta    = typeof StackExchange.options.site.parentUrl !== 'undefined';
+const parentUrl = StackExchange.options.site.parentUrl || `https://${location.hostname}`;
+const metaUrl   = StackExchange.options.site.childUrl;
+
+// Add your user ID here (or set the corresponding value in your Local Storage) to promote yourself
+// to a "superuser", which enables rarely-used options and decreases the number of confirmations.
+const superusers  = [584192, 366904, 6451573];
+const isSuperuser = superusers.includes(StackExchange.options.user.userId) ||
+                    ((localStorage.getItem('samliew-aipmm.isSuperuser') ?? 'false') === 'true');
 
-const newlines = '\n\n';
-const fkey = StackExchange.options.user.fkey;
-const getQueryParam = key => new URLSearchParams(window.location.search).get(key) || '';
-const isSO = location.hostname == 'stackoverflow.com';
-const isSOMeta = location.hostname == 'meta.stackoverflow.com';
-const isMeta = typeof StackExchange.options.site.parentUrl !== 'undefined';
-const parentUrl = StackExchange.options.site.parentUrl || 'https://' + location.hostname;
-const metaUrl = StackExchange.options.site.childUrl;
+// This option defaults to "false". Manually set it to "true" (or set the corresponding value
+// in your Local Storage) to allow destroying spam accounts as fast as possible
+// without multiple confirmations.
+const underSpamAttackMode = ((localStorage.getItem('samliew-aipmm.underSpamAttackMode') ?? 'false') === 'true');
```

Comment:
Where is this LS value being set? Originally was `isSuperuser || false`.
Can we change the key to `SOMU-` instead of `samliew-`?

Reply:
It's not set anywhere. The idea is that the user of the script *can* set it (manually), instead of editing the default in the code. This is better because it doesn't interfere with automatic updates of the script like editing the code does.